pint.json

```diff
@@ -0,0 +1,3 @@
+{
+  "preset": "laravel"
```

Comment:
Would it be possible to use this setting:

```json
{
    "preset": "laravel",
    "rules": {
        "method_argument_space": true,
        "multiline_whitespace_before_semicolons": {
            "strategy": "new_line_for_chained_calls"
        },
        "types_spaces": {
            "space": "single"
        },
        "concat_space": false
    }
}
```

Since that's what we use in all our packages.